src/pages/home.html

```diff
@@ -140,3 +143,57 @@ <h2>How can you help?</h2>
 ## Blog Posts
 Here are the most recent posts we have published on our blog:
 -->
+
+
+<script>
```

Comment:
I don't think you need this `<script />` in `home.html` anymore now that you have it in `base.njk` ?

Reply:
Yes! Sorry, I've forgotten to remove it once again apparently :(